src/main/java/ro/itschool/ema/models/entities/Address.java

```diff
@@ -0,0 +1,38 @@
+package ro.itschool.ema.models.entities;
+
+import lombok.Data;
+import jakarta.persistence.*;
+@Data
```

Comment:
This is not recommended on entities. remove it and use other annotations

Reply:
Solved for Address and Participant entity.